deepfence_server/reporters/graph/threat_graph.go

```diff
@@ -65,7 +65,7 @@ func (tc *ThreatGraphReporter) GetThreatGraph(ctx context.Context, filter Threat
 					VulnerabilityCount:              nodeInfo[index].VulnerabilityCount,
 					ExploitableVulnerabilitiesCount: nodeInfo[index].ExploitableVulnerabilitiesCount,
 					SecretsCount:                    nodeInfo[index].SecretsCount,
-					ExploitableSecretsCount:         nodeInfo[index].ExploitableSecretsCount,
+					ExploitableSecretCount:          nodeInfo[index].ExploitableSecretCount,
```

Comment:
Why removing plural here?

Reply:
I have changed the field name in the DB to `exploitable_secret_count`. 
Hence made the name change for this field. 
Let me know if we should go with `exploitable_secrets_count` instead?

Reply:
Changed to use plurals